corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -41,6 +41,10 @@ FormplayerFrontend.module("Menus", function (Menus, FormplayerFrontend, Backbone
                 }
 
                 Menus.Controller.showMenu(menuResponse);
+                // If menuResponse is an entities type, make set search bar continues to show search
+                if (menuResponse.type == "entities") {
```

Comment:
Would this work if you instead populated `searchText` whenever `urlObject.search` has a value? I think that would be more future-proof in the event that we add other types of search.

There is already one other type of search in web apps ("case claim", which allows you to search all cases in the domain instead of just the cases that your user "owns").